source/packages/libraries/clients/assetlibrary-client/src/client/templates.apigw.service.ts

```diff
@@ -64,7 +64,12 @@ export class TemplatesApigwService extends TemplatesServiceBase implements Templ
         ow(resource.templateId,'templateId', ow.string.nonEmpty);
         ow(resource.category,'category', ow.string.nonEmpty);
 
-        const url = `${this.baseUrl}${super.templateRelativeUrl(resource.category, resource.templateId)}`;
```

Comment:
What is the reasoning behind this change ?!  I dont see any functionality change

Reply:
Same comment - no real functionality change here, but it is consistent with the `createTemplate` functionality. When developing, it was a little confusing when creating/updating templates to see one delete your `resource.templateId`/`resource.category`, but the other one did not.